projects/docs/docs/commands/cache.md

```diff
@@ -52,3 +53,4 @@ tuist cache print-hashes
 | `--path`              | `-p`  | The path to the directory that contains the definition of the project.                                            | Current directory     | No       |
 | `--profile`           | `-P`  | The name of the profile to be used when warming up the cache.                                                     |                       | No       |
 | `--xcframeworks`      | `-x`  | When passed it builds xcframeworks (simulator and device) instead of frameworks (only simulator).                 | False                 | No       |
+| `--xcframeworks-type` | `N/A` | Output type of xcframeworks when `--xcframeworks` is passed (device/simulator)                                    | Null                  | No       |
```

Comment:
I understand from the implementation point of view it's `Null` but from the consumer point of view, it's imho "blank"

```suggestion
| `--xcframeworks-type` | `N/A` | Output type of xcframeworks when `--xcframeworks` is passed (device/simulator)                                    |                      | No       |
```